extensions/wikia/PortableInfobox/js/spec/PortableInfobox.spec.js

```diff
@@ -0,0 +1,51 @@
+/* global beforeEach, describe, it, expect, jasmine, mw, $ */
+
+describe('Portable Infobox', function () {
+	describe('collapsible sections', function () {
+		var portableInfoboxHtml = `
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Replace this ES6 template string with string concatenation [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-js-rules%3AStringInterpolation)

[project-id]: wikia:app